packages/babel-generator/test/preserve-format.js

```diff
@@ -14,6 +14,10 @@ const { __dirname } = commonJS(import.meta.url);
 const suites = (fixtures.default || fixtures)(path.join(__dirname, "fixtures"));
 
 const FAILURES = [
+  // Todo: support trailing comments spanned across trailing comma
+  "import-phases/source-expression-comments/input.js",
+  "import-phases/source-expression-options-comments/input.js",
+
```

Comment:
Note: this is a new issue and should be fixed in another PR.

Reply:
Oh they are marked as trailing comments now due to https://github.com/babel/babel/blob/205e1cfd46407616a389a8a059316c3f2b0f1b95/packages/babel-parser/src/parser/comments.ts#L174-L176

If the `experimental_preserveFormat` works well, maybe we can get rid of this part in Babel 8?

Reply:
Yes we could try